.github/workflows/build-wheel-linux-arm64.yaml

```diff
@@ -0,0 +1,418 @@
+        container_img: ["quay.io/pypa/manylinux_2_28_aarch64"]
+        container_name: ["manylinux_2_28_aarch64"]
+
+    name: Build Wheels (Python ${{ matrix.python_version }})
+    runs-on:
+      group: 'Office 24th floor M2 Mac'
+    
+    continue-on-error: true
+
+    steps:
+    - name: Checkout Catalyst repo
+      uses: actions/checkout@v4
+
+    - name: Setup Runner Environment
+      id: setup_env
+      uses: ./.github/workflows/utils/setup_self_hosted_macos_env
+      with:
+        python_version: ${{ matrix.python_version }}
+
+    - name: Setup Colima VM
+      run: |
+        set -x
+        colima stop
+        colima start --cpu 4 --memory 16
+
+    - name: Get Cached LLVM Source
+      id: cache-llvm-source
+      uses: actions/cache@v4
+      with:
+        path: mlir/llvm-project
+        key: ${{ runner.os }}-${{ matrix.container_name }}-llvm-${{ needs.constants.outputs.llvm_version }}-generic-source
+        enableCrossOsArchive: True
+        fail-on-cache-miss: True
+
+    - name: Get Cached LLVM Build
+      id: cache-llvm-build
+      uses: actions/cache@v4
+      with:
+        path:  llvm-build
+        key: ${{ runner.os }}-${{ runner.arch }}-${{ matrix.container_name }}-llvm-${{ needs.constants.outputs.llvm_version }}-${{matrix.python_version}}-generic-build
+        fail-on-cache-miss: True
+
+    - name: Get Cached MHLO Source
+      id: cache-mhlo-source
+      uses: actions/cache@v4
+      with:
+        path: mlir/mlir-hlo
+        key: ${{ runner.os }}-${{ matrix.container_name }}-mhlo-${{ needs.constants.outputs.mhlo_version }}-generic-source
+        enableCrossOsArchive: True
+        fail-on-cache-miss: True
+
+    - name: Get Cached MHLO Build
+      id: cache-mhlo-build
+      uses: actions/cache@v4
+      with:
+        path: mhlo-build
+        key: ${{ runner.os }}-${{ runner.arch }}-${{ matrix.container_name }}-mhlo-${{ needs.constants.outputs.mhlo_version }}-generic-build
+        fail-on-cache-miss: True
+
+    - name: Get Cached Enzyme Source
+      id: cache-enzyme-source
+      uses: actions/cache@v4
+      with:
+        path: mlir/Enzyme
+        key: ${{ runner.os }}-${{ matrix.container_name }}-enzyme-${{ needs.constants.outputs.enzyme_version }}-generic-source
+        fail-on-cache-miss: True
+
+    - name: Get Cached Enzyme Build
+      id: cache-enzyme-build
+      uses: actions/cache@v4
+      with:
+        path: enzyme-build
+        key: ${{ runner.os }}-${{ runner.arch }}-${{ matrix.container_name }}-enzyme-${{ needs.constants.outputs.llvm_version }}-${{ needs.constants.outputs.enzyme_version }}-generic-build
+        fail-on-cache-miss: True
+
+    # Build Catalyst Wheel
+    - name: Build Catalyst Wheel
+      run: |
+        set -x
+        docker run --platform linux/aarch64 \
+            -v /var/run/docker.sock:/var/run/docker.sock \
+            -v `pwd`:/catalyst \
+            -i ${{ matrix.container_img }} \
+            bash -c "   set -x && \
```

Comment:
let me suggest using `-e` flag. It will make shell exit after first error. So no trailing `&&` would be needed:
``` sh
set -e -x
cmd1 
cmd2
...
cmd_that_can_fail || true
...
cmdN
```

Reply:
Awesome! Done